Gemfile.lock

```diff
@@ -396,7 +400,7 @@ GEM
       concurrent-ruby (~> 1.0)
     inflection (1.0.0)
     interception (0.5)
-    jenkins_api_client (2.0.0)
+    jenkins_api_client (2.1.0)
```

Comment:
### [jenkins_api_client](https://github.com/arangamani/jenkins_api_client)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.0.0 → 2.1.0

[_~~change-log~~, [source-code](https://github.com/arangamani/jenkins_api_client), [gem-diff](https://my.diffend.io/gems/jenkins_api_client/2.0.0/2.1.0)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/arangamani/jenkins_api_client/compare/v2.0.0...v2.1.0).

These are the individual commits:
- (cbed0d8) [Switch to new as we'll update http instance after this](https://github.com/arangamani/jenkins_api_client/commit/cbed0d8964c7b60cc1025b9be6852002fda32d74)
- (55be1b7) [Tweak spelling and grammar in Readme](https://github.com/arangamani/jenkins_api_client/commit/55be1b7ff837d9318dbb1443a4df19580868ea18)
- (9c0733b) [Merge branch 'master' into patch-1](https://github.com/arangamani/jenkins_api_client/commit/9c0733b84044c6ec669d38fab762dd188b9c9cc1)
- (ab234f2) [Merge pull request #280 from libin/master](https://github.com/arangamani/jenkins_api_client/commit/ab234f210933aacc6f523b7503f0b2077c629ddb)
- (3fffb8b) [Merge pull request #284 from bensaunders/patch-1](https://github.com/arangamani/jenkins_api_client/commit/3fffb8b4c994f48fe388afc804cba12662c4d404)
- (7774570) [allow more than 100 levels of nesting](https://github.com/arangamani/jenkins_api_client/commit/777457026863cedca3d54c268f013407c8faf2e3)
- (a146c91) [be more explicit](https://github.com/arangamani/jenkins_api_client/commit/a146c91a7067f6b56333c0550c1c8bf643666417)
- (73f9c52) [Merge pull request #318 from carlosdem/master](https://github.com/arangamani/jenkins_api_client/commit/73f9c52b00bef9055bc4c79406caeb74cb694337)
- (5bdccee) [test for nesting depth](https://github.com/arangamani/jenkins_api_client/commit/5bdccee72512ffe6e339ab706bd4405c2dbb8e07)
- (1b59f19) [v2.1.0](https://github.com/arangamani/jenkins_api_client/commit/1b59f19a1c97523d3d7b815b8fcb2d67510dfe52)


</details>